### ISSUE TITLE
IPC stopped working 

### ISSUE THREAD

**Issue by ang-st** [ava-labs/avalanchego#304]
**Describe the bug**
While monitoring transaction through IPC on one of my node, it stopped to stream at some point.
Upon restarting, the node is now having hard time to resync

**Operating System**

Linux / Docker 
```
└─[$] <git:(master*)> docker version 
Client:
 Version:           18.09.7
 API version:       1.39
 Go version:        go1.10.1
 Git commit:        2d0083d
 Built:             Fri Aug 16 14:20:06 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.09.7
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.1
  Git commit:       2d0083d
  Built:            Wed Aug 14 19:41:23 2019
  OS/Arch:          linux/amd64
  Experimental:     false


└─[$] <git:(master*)> cat /proc/version
Linux version 4.15.0-112-generic (buildd@lcy01-amd64-027) (gcc version 7.5.0 (Ubuntu 7.5.0-3ubuntu1~18.04)) #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020
```
**Additional context**

It started yesterday evening (europe TZ) as i noticed that I did not get notified of some faucet transaction i requested.
Some (at least one as i yet have to check all my utxos)  still don't show up in explorer

https://www.avascan.info/blockchain/x-chain/tx/7vLofdVopKY2QXD8Wtxt2kXZiGpdDMPfP5u9Le7MGCAtm9peL
https://explorer.avax.network/search?query=7vLofdVopKY2QXD8Wtxt2kXZiGpdDMPfP5u9Le7MGCAtm9peL

```
curl -X POST --data '{
    "jsonrpc":"2.0",
    "id"     :1,
    "method" :"avm.getTx",
    "params" :{
        "txID":"7vLofdVopKY2QXD8Wtxt2kXZiGpdDMPfP5u9Le7MGCAtm9peL"
    }
}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/bc/X
{"jsonrpc":"2.0","result":{"tx":"11111113ZyBfsX9JQnBvCJ6sUMwb8dd6o2ya4zLiqzVYyiGEmkXT8PfcNgNpbJeGJkpBQ3jZBgJBjPohqtUX1BLPpZFkGWvpaHRdLHoS2CPezmf88SrCfCJzMhqCiK82rhkDnusAUsKyMCudVKXGmaTeAPGRvGHW5tdYRqxCMg7J4yXBCHgsT4LkMPQ2wbEjXiMbwUDmV6iVQUzDbTp3LREjyreUE3Y61jVhAtrRmfWhnhSt1xp4ijS29gpsie16wu2ib9vyj5SKxCfEbnUzoiAvQvVfxYE83LhAvPXzZU2spCNt55mv92gZFR3e6489mm2R443txnafAHvAY1buDYUL62jqjtrQTkQwTWk5wt9jXcSX2dqGMp2fpQCZTSRRrRNNu3QdKXbN1aviTAvGWcKkNf3z2yi6VQuW71mUjgT4FTomXbcmprRZ9XSqij5sLLKQ63gZ7BETxRDDmVTrVCR2T7xKMwoLsU4262CmnxmpNWVcqgb9K3ZyCbGPs3gv"},"id":1}
root@0998a0197214:/go/src/github.com/ava-labs/gecko# 
```

By submitting this issue I agree to the [Terms and Conditions](https://files.avalabs.org/legal/bug_hunt_q2_2020_terms.pdf) of the [Developer Accelerator Program](https://avalabs.org/dap).

**Comment by danlaine** [comment]
Hey @tyler-smith did we resolve this?

**Comment by tyler-smith** [comment]
Socket issues should be addressed in #360

**Comment by danlaine** [comment]
@ang-st the above PR has been merged. Thanks for reporting and let us know if you have any other issues.
